x-pack/plugin/src/test/resources/rest-api-spec/api/ilm.delete_lifecycle.json

```diff
@@ -1,6 +1,6 @@
 {
   "ilm.delete_lifecycle": {
-    "documentation": "http://www.elastic.co/guide/en/index_lifecycle/current/index_lifecycle.html",
+    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/current/ilm-delete-lifecycle.html",
```

Comment:
I'm not sure what is being recommend here. I think the intention of using `current` here is to prevent the need to update this all the time between versions, and the assumption is these are valid in the branch that it is merged into